openamp/overview.rst

```diff
@@ -137,9 +194,12 @@ The components comprising OpenAMP are:
     :ref:`Libmetal<overview-proxy-libmetal-label>`, Hardware Abstraction
 
 
-The :ref:`topology<topology-work-label>` is limited to main controller to remote system but otherwise open to the implementation.
+The :ref:`topology<topology-work-label>` is limited to main controller to remote system but otherwise
+open to the implementation.
 
-The architecture is exemplified below via a daisy chained topology, with the center processor being both remote and main controller for the next processor in the chain. This is an alternate topology to the previous example in the :ref:`OpenAMP Fundamentals<openamp-fundamentals-work-label>` section.
+The architecture is exemplified below via a daisy chained topology, with the center processor being
+both remote and main controller for the next processor in the chain. This is an alternate topology
+to the previous example in the :ref:`OpenAMP Fundamentals<openamp-fundamentals-work-label>` section.
 
 ..  image::  ../images/architecture/overview-architecture.svg
```

Comment:
It's still unclear how popular this chained architecture is. While possible, I haven't encountered it a lot - one host and one remote seem to be the golden standard.

Let's  align to https://openamp--54.org.readthedocs.build/en/54/protocol_details/components.html#topologies which only shows 1 host + 1 remote.

Reply:
@wmamills  has entered a related issue https://github.com/OpenAMP/openamp-docs/issues/50
 I don't think it is blocking for this release but to be addressed in a separate PR